### DISCUSSION TITLE
Citation key. How to prevent to use "editor(s)"

### DISCUSSION THREAD

**Discussion by Mr-Niko** [retorquere/zotero-better-bibtex#2259] [Q&A] [answered]
`auth.lower + shorttitle(3,3) + year`

If _Author_ field EMPTY → function `auth.lower` use _Editor_ field

## Are there easy way to skip (not to use) _Editor_ if _Author_ is EMPTY?

- as stated on [JabRef doc](https://docs.jabref.org/setup/citationkeypatterns) they have `[pureauth]`, or `[pureauthors3]` in the case but seems to me missed the same alternative on [better-bibtex doc](https://retorque.re/zotero-better-bibtex/citing/), right?
- maybe there is a way to use **regex**?

Thanks!

**Comment by retorquere** [comment, marked as answer]
Not currently possible, but if you open an issue I can add it.
